docs/source/ja/_toctree.yml

```diff
@@ -245,6 +245,505 @@
     - local: main_classes/image_processor
       title: 画像処理プロセッサ
     title: 主要なクラス
+  - sections:
+    - isExpanded: false
+      sections:
+      - local: model_doc/albert
+        title: ALBERT
+      # - local: model_doc/bart
```

Comment:
Not exactly sure why the doc-builder preview link is not working, but can you maybe try removing all the untranslated models from the `toctree` and see if that helps?

Reply:
OK! But it's possible that an indentation error in L102 may have affected it, so let me just check that first.

Reply:
Also, I got a 500 Error in [https://huggingface.co/docs/transformers/v4.35.1/ja/index](https://huggingface.co/docs/transformers/v4.35.1/ja/index)

Reply:
I'm able to preview the docs [here](https://moon-ci-docs.huggingface.co/docs/transformers/pr_27401/ja/model_doc/albert), but I'm not sure why there's a 500 on the main docs. I'll check with the team :)